modals-notifications/es5/src/js/parent.js

```diff
@@ -0,0 +1,23 @@
+function Parent(config) {
+	this.config = config;
+	this.parent = document.createElement('div');
+	this.elem = document.createElement('div');
+}
+
+Parent.prototype = {
+	constructor: Parent,
+
+	show: function () {
+		this.parent.insertAdjacentHTML('afterbegin', this.html);
+
+		this._afterRender()
+	},
+
+	_hide: function () {
+		this.elem.remove();
+	},
+
+	_afterRender: function () {
```

Comment:
You can add a subscription on X click here, as both components share similar logic